src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -608,6 +608,39 @@ namespace UtilityRoutines {
         return FuelTypeErrorsFound;
     }
 
+    Real64 epElapsedTime()
```

Comment:
Moved the epElapsedTime function to UtilityRoutines.  I think this is worth a second pass to clean up the Objexx timing functions, which could lead to trimming down the Objexx library even further.  But that would muddy this PR up, so that can just be :pushpin: